src/index.js

```diff
@@ -75,6 +75,13 @@ function initServer () {
     req.query = _.extend(req.query || {}, req.body || {})
     next()
   })
+  // Trim trailing/leading whitespace from string param values;  
```

Comment:
I'm cool with this going into core, but you should also pull in this logic and delete it from featureserver:
https://github.com/koopjs/FeatureServer/blob/master/src/route.js#L87-L105